arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1777,6 +1696,9 @@
       }
     },
     "EndpointProperties": {
```

Comment:
This model should inherit from "EndpointPropertiesUpdateParameters". This will avoid duplication of properties.